Examples/BookingSystem.AspNetCore/Stores/OrderStore.cs

```diff
@@ -63,13 +63,23 @@ public override void TriggerTestAction(OpenBookingSimulateAction simulateAction,
                         throw new OpenBookingException(new UnknownOrderError());
                     }
                     break;
+                case SellerRequestedCancellationWithMessageSimulateAction _:
+                    if (idComponents.OrderType != OrderType.Order)
+                    {
+                        throw new OpenBookingException(new UnexpectedOrderTypeError(), "Expected Order");
+                    }
+                    if (!FakeBookingSystem.Database.CancelOrderItems(null, null, idComponents.uuid, null, false))
+                    {
+                        throw new OpenBookingException(new UnknownOrderError());
+                    }
+                    break;
+                    
                 case SellerRequestedCancellationSimulateAction _:
                     if (idComponents.OrderType != OrderType.Order)
                     {
                         throw new OpenBookingException(new UnexpectedOrderTypeError(), "Expected Order");
                     }
-
-                    if (!FakeBookingSystem.Database.CancelOrderItems(null, null, idComponents.uuid, null, false ))
+                    if (!FakeBookingSystem.Database.CancelOrderItems(null, null, idComponents.uuid, null, false))
```

Comment:
Note this line _should not_ create a message

Reply:
It won't, I added new parameter (includeCancellationMessage ) which will be false by default

Reply:
Ah sorry that was with reference to this: https://github.com/openactive/OpenActive.Server.NET/pull/42#discussion_r506584471

Reply:
I am a bit confused now, should I change something more?
this original comment is still true, this line shouldn't create message as it is SellerRequestedCancellationSimulateAction I just posted comment for visibility :)

#42 [(comment)](https://github.com/openactive/OpenActive.Server.NET/pull/42#discussion_r506584471) 
both comments are solved by introducing that new variable, includeCancellationMessage which will only be true in SellerRequestedCancellationWithMessageSimulateAction.